content/authors/chanratty/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+# Display name
+title: Chantelle Hanratty
+
+# Username (this should match your github username)
+authors:
+- chanratty
+
+superuser: false
+
+# Role/position e.g. Graduate student
+role: Recent Master of Science Graduate
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Alberta
+  url: https://www.ualberta.ca/index.html
+
+# A list of your interests, add as many as you want
+interests:
+- Toric Geometry
+- Category Theory
+- Mathematics Education
+- Mathematics Outreach
+
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: Master of Science, Mathematics
+    institution: University of Alberta
+  
```

Comment:
Can you remove this empty line (just edit the file and hit commit). YAML cares about whitespace and when you have a newline it thinks you are about to start a new list.

Reply:
Thank you for your feedback! I believe I have fixed it now.